.ci_support/linux_python2.7.yaml

```diff
@@ -13,31 +13,31 @@ docker_image:
 expat:
 - '2.2'
 gdal:
-- '2.2'
+- '2.3'
 geos:
-- 3.6.2
+- 3.7.0
```

Comment:
We may need to force this and others to older version to get vc9 working.
(Probably geos and hdf5, maybe proj.4 but I haven't checked.)

Reply:
Yeah, looks like in my VM environment geos should be 3.6.2. I think hdf5 has to be 1.10.3 to get the most recent libnetcdf.